pkg/plugin/util/testdata/exampleFile.txt

```diff
@@ -1 +1 @@
-exampleTargetexampleCodeexampleCode
\ No newline at end of file
+exampleTargetexampleCodeexampleCodeexampleCode
```

Comment:
```suggestion
exampleTargetexampleCodeexampleCode
```

It has been updated via the script. 
It is something that we could fix/improve
This file just need to have any text for the test